cmd/create.go

```diff
@@ -52,7 +48,11 @@ to quickly create a Cobra application.`,
 			log.Panic(err)
 		}
 
+		// todo:
+		// isolate commands, in case we want to run some validations on the create, it would be a good idea to call the
+		// functions that does the validations
 		infoCmd.Run(cmd, args)
+
```

Comment:
is the screen locking the terminal?

Reply:
not anymore, in this PR there is an update to use a non blocker call.